api/openapi.yaml

```diff
@@ -4657,7 +4693,7 @@ paths:
         content:
           application/json:
             schema:
-              $ref: '#/components/schemas/RegionFormData'
+              $ref: '#/components/schemas/Region'
```

Comment:
This is a breaking change introduced by https://phorge.dev.yugabyte.com/D26078, that should be reverted.